tasks/sep/021-refresh-signers-fakeSC/README.md

```diff
@@ -0,0 +1,30 @@
+# Sepolia Update Signers `FakeSC` and `FakeFoS` and `FakeFuS`
+
+Status: [READY TO SIGN]
+
+## Objective
+
+This task **update** the signers of the `FakeSC`(`0xf64bc17485f0B4Ea5F06A96514182FC4cB561977`) and `FakeFuS`(`0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B`) and `FakeFoS`(`0x837DE453AD5F21E89771e3c06239d8236c0EFd5E`) on Sepolia.
```

Comment:
Seems this is not right, we only update it for one safe here?

Reply:
You are correct, I thought I forgot to update this task after multiples copy paste. 
Change addressed here.